package.json

```diff
@@ -78,6 +83,7 @@
     "mockdate": "^2.0.5",
     "node-sass": "^4.12.0",
     "prettier": "^1.18.2",
+    "rimraf": "^3.0.2",
```

Comment:
Added `rimraf` to support directory cleaning. We could implement this for each of the build/dist dirs.